docs/Developers/Development_Setup.rst

```diff
@@ -58,11 +58,20 @@ in your current directory, as coala only works for Python >= 3.4.4
     $ virtualenv coala-venv -p $(which python3)
 
 - Run coala-venv :
+    + If you are a Linux or Mac user:
+
     ::
 
         $ source coala-venv/bin/activate
         (coala-venv)$ deactivate # to exit the environment
 
+    + And if you are a Windows user:
+
+    ::
+
+        > coala-venv\Scripts\activate.bat
```

Comment:
I don't have a windows sytem, but https://virtualenv.pypa.io/en/stable/userguide/#activate-script says the command should be `> Scripts\activate` and Windows will use either activate.bat or activate.ps1 (as appropriate) to activate the virtual environment.

Reply:
Alright. Should I also add the changing execution policy part for the powershell @li-boxuan ?

Reply:
Maybe you could add a link reference here.